docs/zh/latest/plugins/datadog.md

```diff
@@ -0,0 +1,145 @@
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 简介
+
+`datadog` 是一个内置于 Apache APISIX 的监控插件，可与 [Datadog](https://www.datadoghq.com/)（云计算应用最常用的监控和观察平台之一）无缝集成。如果启用，这个插件支持对每个请求和响应周期进行多种指标参数的获取，这些指标参数基本上反映了系统的行为和健康状况。
+
+该插件通过 UDP 协议将其自定义指标推送给 DogStatsD 服务器，该服务器与 Datadog 代理捆绑在一起（要了解更多关于如何安装 Datadog 代理，请访问[这里](https://docs.datadoghq.com/agent/) ）。DogStatsD 基本上是 StatsD 协议的一个实现，它为 Apache APISIX 代理收集自定义指标，并将其聚合成一个数据点，发送到配置的 Datadog 服务器。要了解更多关于 DogStatsD 的信息，请访问 [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent) 文档。
+
+这个插件具备将多个指标组成一个批处理，一起推送给外部 Datadog 代理的能力，且重复使用同一个数据报套接字。如果你没有收到日志数据，不要担心，给它一些时间。在我们批处理程序中的定时器功能到期后，它会自动发送日志。
+
+关于 Apache APISIX 的批处理程序的更多信息，请参考 [Batch-Processor](../batch-processor.md#配置)
+
+## 属性
+
+| 名称             | 类型   | 必选项  | 默认值      | 有效值       | 描述                                                                                |
+| -----------      | ------ | -----------  | -------      | -----       | ------------------------------------------------------------                               |
+| prefer_name      | boolean | optional    | true         | true/false  | 如果设置为 "false"，将使用路由/服务的 ID，而不是带有度量标签的名称（默认）。   |
+
+该插件支持使用批处理程序来聚集和处理条目（日志/数据）的批次。这就避免了插件频繁地提交数据，默认情况下，批处理程序每 `5` 秒或当队列中的数据达到 `1000` 时提交数据。有关信息或自定义批处理程序的参数设置，请参阅[批处理程序](../batch-processor.md#configuration) 配置部分。
+
+## 元数据
+
+| 名称        | 类型    | 必选项 |     默认值        | 有效值         | 描述                                                            |
+| ----------- | ------  | ----------- |      -------       | -----         | ---------------------------------------------------------------------- |
+| host        | string  | optional    |  "127.0.0.1"       |               | DogStatsD 服务器的主机地址                                      |
+| port        | integer | optional    |    8125            |               | DogStatsD服务器的主机端口                                         |
+| namespace   | string  | optional    |    "apisix"        |               | 由APISIX代理发送的所有自定义度量的前缀。对寻找指标图的实体很有帮助，例如：(apisix.request.counter)。                                        |
+| constant_tags | array | optional    | [ "source:apisix" ] |              | 嵌入到生成指标中的静态标签。这对某些信号度量进行分组很有用。 |
+
+要了解更多关于如何有效地编写标签，请访问[这里](https://docs.datadoghq.com/getting_started/tagging/#defining-tags)
+
+## 输出指标
+
+Apache APISIX 代理，对于每个请求响应周期，如果启用了 datadog 插件，就会向 DogStatsD 服务器输出以下指标。
+
+| Metric Name               | StatsD Type   | Description               |
+| -----------               | -----------   | -------                   |
+| Request Counter           | Counter       | No of requests received.   |
```

Comment:
I am waiting for the exact explanation of the original author @bisakhmondal  here. 
Please do not review it for the time being.

Reply:
Excellent explanation, and my deepest thanks to you!
In fact, my puzzle is `No`.
And your explanation meets my want.
And then I'm going to change `No` to `Number`.
Thanks again.